extension/httpfs/httpfs.cpp

```diff
@@ -87,19 +87,14 @@ RunRequestWithRetry(const std::function<duckdb_httplib_openssl::Result(void)> &r
 	while (true) {
 		auto res = request();
 
-		// Connection level errors to retry
-		bool should_retry = res.error() == duckdb_httplib_openssl::Error::Read;
-		should_retry = should_retry || res.error() == duckdb_httplib_openssl::Error::Connection;
-		should_retry = should_retry || res.error() == duckdb_httplib_openssl::Error::Write;
-
+		// Note: all duckdb_httplib_openssl::Error types will be retried.
```

Comment:
Should we not add additional cases to the switch to achieve this? Otherwise we are still returning the response immediately, no?

Reply:
Thanks - makes sense